[scroll-animations-1] Removed fill from ScrollTimeline

JTensai opened the pull request:
Removed fill from ScrollTimeline in favor of returning Infinity values for before start and after end states of the timeline.

Also fixed edit link by correcting shortname. 

Related Issues:
https://github.com/w3c/csswg-drafts/issues/2066 (Discussion for change)
https://github.com/w3c/csswg-drafts/issues/4347 (Solved by this change)

majido commented on scroll-animations-1/Overview.bs:
```diff
@@ -7,13 +7,14 @@ Work Status: exploring
 Editor: Botond Ballo <botond@mozilla.com>
 Editor: Stephen McGruer <smcgruer@google.com>
 Editor: Antoine Quint <graouts@apple.com>
+Editor: Jordan Taylor <jortaylo@microsoft.com>
```
FYI you don't need to be an editor to make this change.

https://github.com/w3c/csswg-drafts/blob/master/CONTRIBUTING.md#after-discussion

majido commented on scroll-animations-1/Overview.bs:
```diff
@@ -542,25 +508,9 @@ as follows:
 
-1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return an unresolved
-    time value if {{fill}} is <code>none</code> or <code>forwards</code>, or 0 otherwise.
+1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return -Infinity.
```
So if I understand correctly the output is not fully decided yet. I expect this to change once the discussion is resolved here:
https://github.com/w3c/csswg-drafts/issues/2066#issuecomment-580384210

birtles commented:
I've commented a number of times in #2066 that I'd prefer we avoid infinity values and instead use explicit states for this.

JTensai commented:
I have changed the returned values to 0 and time range since I believe we have settled on that at this point. I also updated the examples as requested.

birtles commented:
> I have changed the returned values to 0 and time range since I believe we have settled on that at this point.

Thank you!

majido commented on scroll-animations-1/Overview.bs:
```diff
@@ -542,25 +509,9 @@ as follows:
 
-1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return an unresolved
-    time value if {{fill}} is <code>none</code> or <code>forwards</code>, or 0 otherwise.
+1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return 0.
```
This is correct. However one thing to note is that in this situation the scroll timeline would be in the "InBefore" state which should cause the animation to be in the same state. So while timeline time is 0 but only animations with "fill:backwards" would be active.

At the moment we cannot specify this since `timeline.state` is not specified. But I think we should add a TODO here.

TODO(4750): Define what is the correct timeline state based on scroll offset.

majido replied:
I summarized my understanding of the current plan here.

https://github.com/w3c/csswg-drafts/issues/4325#issuecomment-585295758
